### PR TITLE
Update Korean translation to f749860

### PR DESCRIPTION
**Pull request by marocchino**
Original change can see in this link: [23c2e19...f749860](https://github.com/sinatra/sinatra/compare/23c2e19...f749860#diff-04c6e90faac2675aa89e2176d2eec7d8) 

@yous review please.

### CODE REVIEW

File: README.ko.md [hunk 1]
```diff
@@ -2757,6 +2758,40 @@ ruby myapp.rb [-h] [-x] [-e ENVIRONMENT] [-p PORT] [-o HOST] [-s HANDLER]
 -x # mutex 잠금 켜기 (기본값은 off)
 ```
 
+### 다중 스레드(Multi-threading)
+
+_[Konstantin의 StackOverflow의 답변][so-answer]에서 가져왔습니다_
```
Comment (yous) [inline README.ko.md]: \_Konstantin의 \[StackOverflow 답변][so-answer]에서 가져왔습니다\_

File: README.ko.md [hunk 2]
```diff
@@ -2757,6 +2758,40 @@ ruby myapp.rb [-h] [-x] [-e ENVIRONMENT] [-p PORT] [-o HOST] [-s HANDLER]
...
+
+시나트라는 동시성 모델을 전혀 사용하지 않지만, Thin, Puma, WEBrick 같은
+기저의 Rack 핸들러(서버)는 사용합니다. 시나트라 자신은 스레드에 안전하므로
+랙 핸들러가 동시성 스레드 모델을 사용한다고해도 문제가 되지는 않습니다.
+이는 서버를 시작할 때, 서버에 따른 정확한 호출 방법을 사용했을 때의
+이야기 입니다. 밑의 예제는 다중 스레드 Thin 서버를 시작하는 방법입니다.
```
Comment (yous) [inline README.ko.md]: 이야기 입니다 → 이야기입니다

**Comment by yous** [comment]
I'm done.

**Comment by marocchino** [comment]
@yous Thank you, friend!

**Comment by kytrinyx** [comment]
Thank you!

**Comment by marocchino** [comment]
:smile:

**Pull request merged**